daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -141,7 +146,9 @@ type Status struct {
 	// UdevSyncSupported is true if sync is supported.
 	UdevSyncSupported bool
 	// DeferredRemoveEnabled is true then the device is not unmounted.
-	DeferredRemoveEnabled bool
+	DeferredRemoveEnabled      bool
```

Comment:
Removal is device deactivation. So it basically removes the device like /dev/dm-1. Deletion is actually removal of thin device from the thin pool. Removal does not delete the device from thin pool and once can activate it again later. But once the deletion is done, device is gone for ever and can't be activated again.